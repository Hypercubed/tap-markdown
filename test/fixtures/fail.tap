TAP version 13
# pass
ok 1 should pass
# fail
not ok 2 should fail
  ---
    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:9:5)
  ...
# pass and fail
ok 3 should pass
not ok 4 should fail
  ---
    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:15:5)
  ...

1..4
# tests 4
# pass  2
# fail  2
