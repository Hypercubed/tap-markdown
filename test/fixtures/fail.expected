# Tests

- ✔ pass [pass: 1, fail: 0]
- ✖ fail [pass: 0, fail: 1]
- ✖ pass and fail [pass: 1, fail: 1]

# Summary

- planned: 4
- assertions: 4
- pass: 2
- fail: 2

# Fails

## fail

```
✖ should fail
operator: ok
expected: true
actual:   false
```

## pass and fail

```
✖ should fail
operator: ok
expected: true
actual:   false
```

